docs/docs/09-typescript/01-resources.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Resources
+id: resources
+---
+
+Resources are created similar to other CDKs (awscdk, cdktf, cdk8s, etc.). They make up the tree that represents your infrastructure.
+Resources are classes typically constructed with at least two arguments: The scope (parent) resource and an id.
+
+The first argument for `main` is the root of this tree of resources and can be used to construct your first resource:
+
+```ts
+import { main, cloud } from "@wingcloud/framework";
+
+main((root) => {
```

Comment:
```suggestion
main(root => {
```

nit: style... (feel free to ignore)

Reply:
I'm going to keep it as-is because this is how our current eslint/prettier config formats this (even with only 1 arg).

IMO I'm also a fan of it because I appreciate the consistency when you have multiple args.